sources/packages/web/src/components/common/AnnouncementBanner.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <banner
+    header="System Announcements"
+    :type="BannerTypes.Warning"
+    v-if="relevantAnnouncements && relevantAnnouncements.length > 0"
+  >
+    <template #content>
+      <div v-for="(announcement, index) in relevantAnnouncements" :key="index">
+        <b>{{ announcement.messageTitle }}</b> -
```

Comment:
Minor. For bold fonts, we were using different font-family once, but now we at least have a distinct variable for font-family when it comes to bold fonts. 

![image](https://github.com/user-attachments/assets/1767cd97-3080-4b03-92ab-499a151fe50e)


I would suggest one of the 2 things. 


1. add the class `font-bold` to the element. 
2. write a generic css

![image](https://github.com/user-attachments/assets/99f34d15-f462-47b8-a56e-95e4a4904388)

Reply:
changed to a span of font-bold as seen elsewhere